lib/matplotlib/tests/test_colors.py

```diff
@@ -1829,3 +1830,52 @@ def test_LinearSegmentedColormap_from_list_value_color_tuple():
         cmap([value for value, _ in value_color_tuples]),
         to_rgba_array([color for _, color in value_color_tuples]),
     )
+
+
+@image_comparison(['test_norm_protocol.png'])
```

Comment:
```suggestion
@image_comparison(['test_norm_protocol.png'], remove_text=True)
```
and no need to manually call `remove_ticks_and_titles`.